functions/src/tebra-sync/__tests__/tebraProxyClient.test.js

```
@@ -0,0 +1,443 @@
+        url: 'https://test-service.run.app/test-endpoint',
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+          'X-API-Key': 'test-api-key'
+        },
+        data: { test: 'data' }
+      });
+    });
+
+    it('should handle request without API key', async () => {
+      delete process.env.TEBRA_INTERNAL_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)